slides/aml-06-linear-models-classification/index.html

```diff
@@ -690,8 +690,8 @@
 # In sckit-learn
 
 - OvO: only SVC
-- OvR: default for all linear models, even LogisticRegression (deprecated)
-- `LogisticRegression(multinomial=True)`
+- OvR: default for all linear models
```

Comment:
shouldn't it now say "except for logistic regression"? calling the binary one OvR is a weirdness of our implementation and should be hidden from users.